Containerfile

```diff
@@ -10,6 +10,7 @@ RUN microdnf -y install \
     nodejs \
     nodejs-npm \
     python3 \
+    python3-devel \
```

Comment:
I added `gcc` explicitly, but we have quite a long list of implicit dependencies (e.g. `make`, `kernel-headers`). I'm not sure what's the correct approach here, though. Making everything explicit will make this list quite long.

Reply:
I know we do. IMO we need to take one for the team and maintain the exploded list, although long, but at least we'd avoid the issue of breaking the container build when any of the packages currently listed updates their SPEC file in such a way where it no longer will transitively pull dependencies we directly depend on.
Let me give you an example from the previous project I worked on: https://gitlab.com/libvirt/libvirt/-/blob/master/ci/containers/debian-11.Dockerfile?ref_type=heads#L13

The list had to be expanded explicitly because the project hit the issue I just described on multiple occasions. Worth noting though that the Dockerfile content was autogenerated completely, including the package list.